documentation/domains/Domain.json

```diff
@@ -618,6 +618,10 @@
           "x-kubernetes-preserve-unknown-fields": "true",
           "description": "The configuration for the WebLogic Monitoring Exporter. If WebLogic Server instances are already running and have the monitoring exporter sidecar container, then changes to this field will be propagated to the exporter without requiring the restart of the WebLogic Server instances.",
           "$ref": "#/definitions/Map"
+        },
+        "port": {
+          "description": "The port exposed by the WebLogic Monitoring Exporter running in the sidecar container. Defaults to 8080 unless that port value is in use by a channel or network access point (NAP) of the WebLogic Server instance, in which case the next higher available port is selected. If a port value is specified then it must not conflict with a port used by the WebLogic Server instance.",
```

Comment:
>  in which case the next higher available port is selected

Is generating a port number heuristically helpful?  Instead perhaps it'd be better just to generate an error message. If I understand correctly, tooling that imports from the exporter must specify this port - so such configuration would need to 'guess' the expected value...

Reply:
I felt compelled to keep it because we've already released with this behavior :(. It's not correct that tooling has to guess the value since the port value is published as one of the Prometheus annotations.